src/screens/User/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+// eslint-disable-next-line import/no-unresolved
+import * as S from './styles';
+import { Id, InfoItem, SubName, SubNameValue, UserId } from './styles';
```

Comment:
Use 1 styles for import
import * as S from './styles'; or import { Id, InfoItem, SubName, SubNameValue, UserId } from './styles';